extensions/amp-story/1.0/amp-story-draggable-drawer.css

```diff
@@ -108,11 +108,6 @@
   transition: none !important;
 }
 
-.i-amphtml-story-draggable-drawer-bookend .i-amphtml-story-draggable-drawer-content {
-  margin-top: auto !important;
```

Comment:
Is a bookend with no configuration still displayed at the bottom of the screen?
I can help you test if you upload a demo :))